modules/apps/adaptive-media/adaptive-media-image-test/src/testIntegration/java/com/liferay/adaptive/media/image/service/impl/test/AMImageEntryLocalServiceTest.java

```diff
@@ -539,8 +539,12 @@ public void testGetPercentageMax100() throws Exception {
 			null;
 
 		try {
+			int defaultSiteInitialImagesCount =
```

Comment:
I thinks is better to call this variables : `expectedAMImageEntriesCount` (or `expectedEntriesCount`) as I have seen on other parts of the Portal